### PR TITLE
refactor: move `electron` files into dedicated folder

### PR DESCRIPTION
**Pull request by afonsojramos**

### CODE REVIEW

File: src/electron/main.js [hunk 1]
```diff
@@ -8,18 +8,12 @@ const {
...
 require('@electron/remote/main').initialize();
 
-const idleIcon = path.join(
-  __dirname,
-  'assets',
-  'images',
-  'tray-idleTemplate.png',
-);
-const activeIcon = path.join(__dirname, 'assets', 'images', 'tray-active.png');
+const idleIcon = `${__dirname}/../../assets/images/tray-idleTemplate.png`;
```
Comment (setchy) [inline src/electron/main.js:15]: From what I've read about electron, this may not work in the packaged version

**Comment by setchy** [comment]
Oh my yes!!! This has been on my to-do list! Love it!!!! ❤️

File: src/electron/main.js (hunk 1 above)
Comment (setchy) [reply src/electron/main.js:15]: Could be a non issue. Worth validating with the dmg

Comment (afonsojramos) [reply src/electron/main.js:15]: Testing now! 🤓

Comment (afonsojramos) [reply src/electron/main.js:15]: It did indeed not work, but I'll make a change to not use join nonetheless

Comment (afonsojramos) [reply src/electron/main.js]: Done! Now working with path.resolve, which is way cleaner than join!

**Review by setchy — APPROVED** [review]

**Pull request merged**